### ISSUE TITLE
About rpn_class_loss

### ISSUE THREAD

**Issue by masonwang96** [matterport/Mask_RCNN#1874]
@waleedka In the rpn_class_loss_graph() function the author writes:

    # Get anchor classes. Convert the -1/+1 match to 0/1 values.
    anchor_class = K.cast(K.equal(rpn_match, 1), tf.int32)
    # Positive and Negative anchors contribute to the loss,
    # but neutral anchors (match value = 0) don't.
    indices = tf.where(K.not_equal(rpn_match, 0))
    # Pick rows that contribute to the loss and filter out the rest.
    rpn_class_logits = tf.gather_nd(rpn_class_logits, indices)
    anchor_class = tf.gather_nd(anchor_class, indices)

However,  I don't see any code corresponding to `Convert the -1/+1 match to 0/1 values.` So the loss is computed on `rpn_class_logits` between 0 and 1 and `rpn_match` from {1, -1}.
Do I miss something? Please help me.
